ipaserver/install/adtrustinstance.py

```diff
@@ -130,6 +130,15 @@ def map_Guests_to_nobody():
     logger.debug("Map BUILTIN\\Guests to a group 'nobody'")
     ipautil.run(args, env=env, raiseonerr=False, capture_error=True)
 
+
+def get_idmap_range(realm):
+    range = api.Command.idrange_show('%s_id_range' % realm)['result']
```

Comment:
``range`` is the name of a builtin function. pylint should complain about this. Could you pick another name, please?

Also let's use ``format()`` everywhere. Bonus points for f-strings if the change doesn't need backporting.

Reply:
It will need a backport, possibly to 4.6. I changed the rest.